Maestro/subscriptions.json

```diff
@@ -825,6 +825,7 @@
         "https://github.com/SignalR/SignalR/blob/master/**/*",
         "https://github.com/SignalR/SignalR/blob/dev/**/*",
         "https://github.com/SignalR/SignalR/blob/release/**/*",
+        "https://github.com/dotnet/arcade-validation/blob/master/**/*",
```

Comment:
Can you alphabetize?  Otherwise fine.

Reply:
I'm apparently blind and didn't realize it was sorted. Fixed